### PR TITLE
ddd,debuild,flux-sched: Fix the build - add missing libraries

### PR DESCRIPTION
**Pull request by bernhardkaindl**
While building lots of autoconf packages to test a PR, these failed on Ubuntu 18.04 due to missing libs

### CODE REVIEW

**Comment by alalazo** [comment]
@spackbot maintainers

**Review by alalazo — APPROVED** [review]

**Comment by alalazo** [comment]
Waiting for @SteVwonder to comment on `libedit` before merging this PR.

**Comment by alalazo** [comment]
@spackbot run pipeline

**Review by SteVwonder — APPROVED** [review]
LGTM! Thanks @bernhardkaindl.

**Pull request merged**
